packages/non_empty_list/non_empty_list.0.1/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+maintainer: "ajo41@cam.ac.uk"
+authors: ["Alistair O'Brien"]
+homepage: "https://github.com/johnyob/ocaml-non-empty-list"
+bug-reports: "https://github.com/johnyob/ocaml-non-empty-list/issues"
+dev-repo: "git+https://github.com/johnyob/ocaml-non-empty-list.git"
+depends: [ 
+  "ocaml" { >= "4.08.0" }
+  "dune" { >= "2.8" }
+  "base" { >= "0.14.1" }
```

Comment:
```suggestion
  "base" { >= "v0.14.1" }
```

Reply:
Can you make this change also on your repo?

Reply:
Sure! Thank you so much for your help. It's my first opam package so, thanks for fixing the lower bound tests :)